openlibrary/plugins/openlibrary/stats.py

```diff
@@ -136,19 +137,11 @@ def register_filter(name, function):
     filters[name] = function
 
 
-def _encode_key_part(key_part):
-    """
-    :param basestring key_part:
-    """
+def _encode_key_part(key_part: str) -> str:
     return key_part.replace('.', '_')
 
 
-def _get_path_page_name(path):
-    """
-    :param str path: url path from e.g. web.ctx.path
-    :rtype: str
-    """
-
+def _get_path_page_name() -> str:
```

Comment:
# ???
```suggestion
def _get_path_page_name(path: str) -> str:
```

Reply:
`path` isn't called within the function, so I thought removing the parameter would help people avoid being confused why passing a specific `path` had no impact on the return value. I hope this is right.